packages/cli/src/commands/org/config/push.ts

```diff
@@ -47,6 +49,12 @@ export default class Push extends Command {
       default: false,
       required: false,
     }),
+    snapshotId: flags.string({
+      char: 's',
+      description:
+        'The unique identifier of the snapshot to push. If not specified, a new snapshot will be created from your local project. You can list available snapshot in your organization with org:config:list',
```

Comment:
```suggestion
        'The unique identifier of the snapshot to push. If not specified, a new snapshot will be created from your local project. You can list available snapshots in your organization with org:config:list',
```

Reply:
I thought snapshot were not updatable 🤔 
What does pushing to a specific snapshotId does?

Reply:
I am not sure about this one. That's why I wanted your opinion on it.
Would it make sense to have a `--snapshotId` flag on all commands expect for the push?
But like you said, I don't see a lot of use cases expect for someone who wants to apply a snapshot that has been created in the Admin.

Reply:
I mean... if there's no use case that we see where someone would need to do that (and that we don't even know what happens when you do it), I would certainly not add the flag for push :D 

> Would it make sense to have a --snapshotId flag on all commands expect for the push?

I think that'd be fine, yes